src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2.Test/ScenarioTests/DataFactoryTests.ps1

```diff
@@ -81,9 +81,7 @@ function Test-CreateDataFactory
         $actual = Set-AzureRmDataFactoryV2 -ResourceGroupName $rgname -Name $dfname -Location $dflocation -Force
```

Comment:
Please add tests for the two new changes.

Reply:
What new changes? I removed the changes for 2 last cmdlets since we don't want them shipped now, and IR tests have been updated.

Reply:
@zhangyd2015

Reply:
Added. However got recording issue in Dogfood environment.